### DISCUSSION TITLE
I can't understand, why should I use this, and not just directly run `pytest` in my workflow yml file

### DISCUSSION THREAD

**Discussion by aahnik** [ymyzk/tox-gh-actions#157] [Q&A] [answered]
My tox.ini looks like this:

```ini
[tox]
isolated_build = True
envlist = py38,py39

[testenv]
deps = pytest
commands = pytest
```

Should I run `pytest` directly in the workflow yml file ?

**Comment by aahnik** [comment]
I think I can also run `tox -e py` to use the default python version.

**Comment by ymyzk** [comment, marked as answer]
You don't need to use this plugin for running pytest or tox.

This is a helper for running tests in parallel on GitHub Actions using multiple jobs. For a simple configuration like what you shared, you may not get benefit of this library.

**Comment by aahnik** [comment]
@ymyzk , thanks a lot for your clarification. 🙏
